examples/wearable/UIComponents/contents/user-inputs-elements/checkbox/checkbox.html

```diff
@@ -1,13 +1,12 @@
-<!DOCTYPE html>
 <html>
 
 <head>
 	<meta content="width=device-width, initial-scale=1, user-scalable=no" name="viewport" />
 	<title>
-		Wearable UI
+		Checkbox
 	</title>
 	<link href="../../../lib/tau/wearable/theme/default/tau.css" rel="stylesheet" />
-	<link href="../../../lib/tau/wearable/theme/default/tau.circle.css" media="all and (-tizen-geometric-shape: circle)" rel="stylesheet" />
```

Comment:
Don't remove the  'media="all and (-tizen-geometric-shape: circle)"'

Reply:
They cause wrong layout on Browser and we already started to get rid of them.

Reply:
TAU currently supports Square and Circle types. If you delete this, it only applies to the Circle, so I don't think it is correct.
@lslachciak, Is it right?

Reply:
we need to discuss this issue

Reply:
Since it's not decided yet I restored media query.